src/backend/sgx/data.rs

```diff
@@ -167,7 +167,7 @@ pub fn dev_sgx_enclave() -> Datum {
 pub fn aesm_socket() -> Datum {
     Datum {
         name: "AESM Daemon Socket".into(),
-        pass: Path::new(AESM_SOCKET).exists(),
+        pass: cfg!(feature = "disable-sgx-attestation") || Path::new(AESM_SOCKET).exists(),
```

Comment:
Can't we do something like this?
```suggestion
        #[cfg(feature = "disable-sgx-attestation")]
        pass: true,
        #[cfg(not(feature = "disable-sgx-attestation"))]
        pass: Path::new(AESM_SOCKET).exists(),
```

Reply:
why make it so complex?